python/TestHarness/TestHarness.py

```diff
@@ -738,6 +738,7 @@ def parseCLArgs(self, argv):
     parser.add_argument('--max-fails', nargs=1, type=int, dest='max_fails', default=50, help='The number of tests allowed to fail before any additional tests will run')
     parser.add_argument('--pbs', nargs='?', metavar='batch_file', dest='pbs', const='generate', help='Enable launching tests via PBS. If no batch file is specified one will be created for you')
     parser.add_argument('--pbs-cleanup', nargs=1, metavar='batch_file', help='Clean up the directories/files created by PBS. You must supply the same batch_file used to launch PBS.')
+    parser.add_argument('--pbs-project', nargs='?', const='moose', default='moose', help='Identify PBS job submission to specified project')
```

Comment:
I am using both a const and default here, to handle the case where the user supplies the --pbs-project argument with out a value.